packages/connectors/package.json

```diff
@@ -45,14 +45,15 @@
     }
   },
   "dependencies": {
-    "@base-org/account": "1.1.1",
+    "@base-org/account": "2.4.0",
     "@coinbase/wallet-sdk": "4.3.6",
     "@gemini-wallet/core": "0.2.0",
     "@metamask/sdk": "0.33.1",
     "@safe-global/safe-apps-provider": "0.18.6",
     "@safe-global/safe-apps-sdk": "9.1.0",
     "@walletconnect/ethereum-provider": "2.21.1",
     "cbw-sdk": "npm:@coinbase/wallet-sdk@3.9.3",
+    "fastestsmallesttextencoderdecoder": "1.0.22",
```

Comment:
```suggestion
```

Reply:
`fastestsmallesttextencoderdecoder` is not a peer dependency of `@base-org/account` (you can check its package.json) so seems like it's the responsibility for `@solana/kit` to set it since a bunch of dependencies it uses (e.g. `@solana/*`) have `fastestsmallesttextencoderdecoder` as a peer dependency.